src/test/scala/de/uhh/lt/jst/dt/WordSimFromCountsSpec.scala

```diff
@@ -0,0 +1,135 @@
+package de.uhh.lt.jst.dt
+
+import com.holdenkarau.spark.testing.{DataFrameSuiteBase, RDDComparisons, Utils}
+import de.uhh.lt.jst.utils.Util
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.DataFrame
+import org.scalatest._
+
+
+/**
+  * Method call graph [with side effects] for system under test (SUT):
+  *
+  * WordSimFromCounts.main [reads file]
+  * +- WordSimLib.computeWordSimsWithFeatures [writes file]
+  *    +- WordSimLib.computeFeatureScores
+  *
+  */
+// TODO: Use synthetic data to be make tests more compact.
+class WordSimFromCountsSpec extends FlatSpec with Matchers with DataFrameSuiteBase with RDDComparisons {
+  type TermTermCountRDD = RDD[(String, (String, Int))]
+  type TermCountRDD =  RDD[(String, Int)]
+  type TermTermScoresRDD = RDD[(String, Array[(String, Double)])]
+
+  val wordsPerFeatureNum: Int = 1000
+  val featuresPerWordNum: Int = 2000
+
+  val wordMinCount: Int = 1
```

Comment:
these default do not correspond the defaults in the bash script and this is VERY important to make them correspond.